src/test/resources/graphql-examples/deposit/addDeposit/addNewDeposit.graphql

```diff
@@ -1,5 +1,5 @@
 mutation {
-    addDeposit(input: {clientMutationId: "my-mutation-id", depositId: "00000000-0000-0000-0000-000000000006", bagName: "bag6", creationTimestamp: "2019-05-15T23:36:00.000Z", depositorId: "archie001"}) {
+    addDeposit(input: {clientMutationId: "my-mutation-id", depositId: "00000000-0000-0000-0000-000000000006", bagName: "bag6", creationTimestamp: "2019-05-15T23:36:00.000Z", depositorId: "archie001", origin: SWORD2}) {
```

Comment:
please add the `origin` in the response query as well (this will also change the expected JSON result)